<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="2" failed="0" ignored="0" total="3" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="Another Skip Suite" duration-ms="66" started-at="2021-07-09T15:54:03 IST" finished-at="2021-07-09T15:54:03 IST">
    <groups>
    </groups>
    <test name="Skip Exception Test Case" duration-ms="66" started-at="2021-07-09T15:54:03 IST" finished-at="2021-07-09T15:54:03 IST">
      <class name="annotations.AnotherSKIPTechniqueTestAnnotation">
        <test-method status="PASS" signature="LoginRediffmail()[pri:1, instance:annotations.AnotherSKIPTechniqueTestAnnotation@614ddd49]" name="LoginRediffmail" duration-ms="15" started-at="2021-07-09T15:54:03 IST" finished-at="2021-07-09T15:54:03 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LoginRediffmail -->
        <test-method status="SKIP" signature="SendMail()[pri:2, instance:annotations.AnotherSKIPTechniqueTestAnnotation@614ddd49]" name="SendMail" duration-ms="2" started-at="2021-07-09T15:54:03 IST" depends-on-methods="annotations.AnotherSKIPTechniqueTestAnnotation.LoginRediffmail" finished-at="2021-07-09T15:54:03 IST">
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Skipping the SendMail test annotation]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Skipping the SendMail test annotation
at annotations.AnotherSKIPTechniqueTestAnnotation.SendMail(AnotherSKIPTechniqueTestAnnotation.java:95)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SendMail -->
        <test-method status="SKIP" signature="Logout()[pri:3, instance:annotations.AnotherSKIPTechniqueTestAnnotation@614ddd49]" name="Logout" duration-ms="0" started-at="2021-07-09T15:54:03 IST" depends-on-methods="annotations.AnotherSKIPTechniqueTestAnnotation.LoginRediffmail, annotations.AnotherSKIPTechniqueTestAnnotation.SendMail" finished-at="2021-07-09T15:54:03 IST">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method AnotherSKIPTechniqueTestAnnotation.Logout()[pri:3, instance:annotations.AnotherSKIPTechniqueTestAnnotation@614ddd49] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method AnotherSKIPTechniqueTestAnnotation.Logout()[pri:3, instance:annotations.AnotherSKIPTechniqueTestAnnotation@614ddd49] depends on not successfully finished methods
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:102)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Logout -->
      </class> <!-- annotations.AnotherSKIPTechniqueTestAnnotation -->
    </test> <!-- Skip Exception Test Case -->
  </suite> <!-- Another Skip Suite -->
</testng-results>
